velox/functions/lib/aggregates/MinMaxByAggregatesBase.h

```diff
@@ -550,7 +560,8 @@ template <
 std::unique_ptr<exec::Aggregate> create(
     TypePtr resultType,
     TypePtr compareType,
-    const std::string& errorMessage) {
+    const std::string& errorMessage,
+    bool throwOnNestedNulls = false) {
```

Comment:
Why do we need this flag here? Is it because this function used by both Presto and Spark and Spark doesn't fail on inputs with nested nulls?

Reply:
Yes, `sparksql::registerMinMaxBy` uses this too, and spark dose not fail on inputs with nested nulls.
```SQL
scala> spark.sql(s"""select min(c1),max(c1) from values (1, array(1,2,3)), (1, array(2,null)) tbl(c0,c1)""").show()+---------+---------+
|  min(c1)|  max(c1)|
+---------+---------+
|[1, 2, 3]|[2, null]|
+---------+---------+


scala> spark.sql(s"""select min_by(c0, c1),max_by(c0, c1) from values (1, array(1,2,3)), (1, array(2,null)) tbl(c0,c1)""").show()
+--------------+--------------+
|min_by(c0, c1)|max_by(c0, c1)|
+--------------+--------------+
|             1|             1|
+--------------+--------------+
```